### PR TITLE
Update docker and k8s plugins to use the new StatusReport helper func.

### PR DESCRIPTION
**Pull request by izaaklauer**
Depends on https://github.com/hashicorp/waypoint-plugin-sdk/pull/55

The new StatusReport function pushes all of this work into the plugin sdk!

### CODE REVIEW

File: internal/cli/status.go [hunk 1]
```diff
@@ -1098,7 +1098,7 @@ their current state, run ‘waypoint status -app=APP-NAME PROJECT-NAME’.
 `)
 
 	wpStatusAppSuccessMsg = strings.TrimSpace(`
-The application and its declared resources listed above represents its current state known
+The application and it's resources listed above represents it's current state known
```
Comment (izaaklauer) [inline internal/cli/status.go:1101]: Totally incidental change -  we're currently displaying actual resources, not declared resources.

**Review by evanphx — APPROVED** [review]

**Review by briancain — APPROVED** [review]
> +8 −117

Nice! :chef_kiss: 👨🏻‍🍳 ❤️

File: internal/cli/status.go (hunk 1 above)
Comment (krantzinator) [reply internal/cli/status.go:1101]: FYI it still would be `its` -- the apostrophe in `it's` does not denote possession, only contraction.
```suggestion
The application and its resources listed above represents its current state known
```

**Review by krantzinator — APPROVED** [review]
Nice!

File: internal/cli/status.go (hunk 1 above)
Comment (izaaklauer) [reply internal/cli/status.go:1101]: 🤦‍♂️🙏

Comment (izaaklauer) [reply internal/cli/status.go:1101]: Fixing in https://github.com/hashicorp/waypoint/pull/2062

**Pull request merged**